ajax/libs/react-chartkick/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "react-chartkick",
+  "description": "Create beautiful JavaScript charts with one line of React",
+  "filename": "react-chartkick.min.js",
+  "version": "0.1.4",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ankane/react-chartkick"
+  },
+  "author": "Andrew Kane",
+  "keywords": [
+    "charts",
+    "graphs"
+  ],
+  "homepage": "https://www.chartkick.com/react",
+  "npmName": "react-chartkick",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "react-chartkick*"
```

Comment:
Let's modify from `"react-chartkick*"` to `"**/*"`.
We can include all the files in `dist` path.(Except there is some file we don't need, such as *.md)
Many thanks.